joomla/joomla-cms
# [4.1] incorrect role on language switcher

Pull Request for Issue #37479 .

### Testing Instructions
Make sure that you test all the variants of the module

the role that was not needed is only for use when you have non-list elements eg div that you want to behave like a select option. you cant apply that to an existing list.
